rector.php

```diff
@@ -17,4 +18,9 @@
         SetList::PHP_74,
         SetList::CODE_QUALITY,
     ]);
+    $rectorConfig->skip([
```

Comment:
rector kept reformatting sdk builder's shutdown handling into a form that didn't work.